easybuild/easyconfigs/r/Rust/Rust-1.70.0-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'Rust'
+version = '1.70.0'
+
+homepage = 'https://www.rust-lang.org'
+description = """Rust is a systems programming language that runs blazingly fast, prevents segfaults,
+ and guarantees thread safety."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://static.rust-lang.org/dist/']
+sources = ['rustc-%(version)s-src.tar.gz']
+patches = ['Rust-1.70_sysroot-fix-interpreter.patch']
+checksums = [
+    {'rustc-1.70.0-src.tar.gz': 'b2bfae000b7a5040e4ec4bbc50a09f21548190cb7570b0ed77358368413bd27c'},
+    {'Rust-1.70_sysroot-fix-interpreter.patch': '72bffa54837d1a2a117ad63c16deb4078a56a3d9d4f4cbc7430d68dd4cb3effc'},
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+    ('Python', '3.11.3'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+configopts = "--enable-extended --sysconfdir=%(installdir)s/etc "
+
+# avoid build dependency on Ninja, which requires Python,
+# since Rust is a build dependency for cryptography that is included with Python
+configopts += "--set=llvm.ninja=false "
```

Comment:
This is no longer true. Should we switch to using Ninja?

Reply:
Yes, that makes sense to me. @Micket?

Reply:
See #18018 for Ninja